src/main/java/ui/issuepanel/PanelControl.java

```diff
@@ -105,6 +113,7 @@ public FilterPanel addPanelAt(int index) {
 
         updatePanelIndices();
         setCurrentlySelectedPanel(Optional.of(index));
+        mainStage.show();
```

Comment:
Alright, that's a good idea. We can document the `NoNodesVisibleException` quirkiness there.